src/app/backend/resource/logs/logs.go

```diff
@@ -37,7 +37,7 @@ type Logs struct {
 	LogViewInfo `json:"logViewInfo"`
 }
 
-// Number that is returned if requested line could not be found
+// LINE_INDEX_NOT_FOUND that is returned if requested line could not be found
```

Comment:
Could you please remove the first 'that':

LINE_INDEX_NOT_FOUND is returned if requested line could not be found

thanks